app/views/govuk_component/docs/button.yml

```diff
@@ -62,3 +62,12 @@ examples:
       text: "I'm a start now button with lots of text to test how the component scales at extremes."
       start: true
       href: "#"
+  with_data_attributes:
+    data:
+      text: "Track this!"
+      margin_bottom: true
+      extra_attrs: {
```

Comment:
There's an existing pattern for passing in data attributes:
https://github.com/alphagov/government-frontend/blob/master/app/views/components/_document-list.html.erb#L13

I wonder how we can document/standardise these patterns.